### DISCUSSION TITLE
（已失效）

### DISCUSSION THREAD

**Discussion by zs55847** [WeNeedHome/SummaryOfLoanSuspension#1081] [General]
已失效

**Comment by ghost** [comment]
中共打不赢美国，难道还整不死你？🤡小屁民

**Reply by blisee** [reply]
中共？一股台湾腔

**Reply by chen961** [reply]
台湾蛙带节奏都不会带

**Reply by NiceAir** [reply]
ghost doesn't have any public repositories yet.

**Reply by 378804441** [reply]
> 中共打不赢美国，难道还整不死你？🤡小屁民

🐸咖喱贡🐒，🐸系呆湾🐺

**Comment by stevenwsg** [comment]
插眼

**Comment by zhw173** [comment]
还是躺平实际点

**Comment by ChinaChenMingQuan** [comment]
你想怂恿我带头冲锋，用血革命，然后你躲在后面享受革命成果？疯了吧你！

**Reply by wedxz** [reply]
> 你想怂恿我带头冲锋，用血革命，然后你躲在后面享受革命成果？疯了吧你！

这还没干啥呢？就开始逗你的小聪明了？

**Reply by chen961** [reply]
@zs55847 怎么一股翻译腔味道，那个中国人会说我在这里道歉？

**Comment by canghaicode** [comment]
现代社会跟古代不一样，古代锄头跟刀枪没有什么武器代差，而现代锄头在枪炮面前就是玩具，而且权力对基层的掌控已达毛细血管

**Reply by IUsers** [reply]
你说得对，然而一个组织最容易崩溃的也恰恰是其内部，苏联如此

**Comment by Kasianian** [comment]
要成为第二个孙中山，首先你要自己的有政治纲领，比如建立民主中国，驱除共匪，恢复中华。初期你会被共产党剿杀，所以你只能到境外例如日本，谋求建立新的中国同盟会，联合海外华人抗共，还得接受资金援助，还有人为革命流血牺牲，直至建立新中华民国，，按目前共产党对舆论和韭民的行踪监控，这一切都会比晚清困难的多。最後要记住，如果你解救了中华民族，不能再让第二个袁世凯窃取革命果实，中华民族几千年受独裁统治的痛苦会不断延续下去。

**Reply by sthfake** [reply]
不要诬蔑袁，宋教仁可能是孙杀的，袁的贡献很大

**Comment by BlackBerryPi** [comment]
时机远没成熟，没有一点武装力量你拿什么当筹码？除非有第三方势力介入，但到时局势就不是我们能掌控的了

**Comment by JayChou55** [comment]
韭菜就安心当韭菜。。。未来一定会发展到，当你有这个想法的时候 就会被监控到 让你进去

**Comment by zs55847** [comment]


**Reply by yellowcaicai** [reply]
哈哈哈，这也太搞笑了

**Reply by Lysice** [reply]
hhhhhhh

**Comment by elespec** [comment]
我认为,这个一呼百应,媒体默许传播的事件,本身就是中央推动的,静观其变就好了;
地方政府的问题,不是一朝一夕能解决的,不要被被恶意带节奏,要相信中央;

**Comment by Pacio** [comment]
有意义嘛？反正这块地无论起义多少次最终的结局都是一样的，循环往复
我觉得还是润出去找个老外改改基因比较靠谱

**Reply by blisee** [reply]
你不配为炎黄子孙

**Reply by ghost** [reply]
人家说的没错啊，你跟个傻逼似的在这叫叫叫，封建专制的国家不润出去，还留在这里干什么？等着被铁拳砸吗？真是你妈的个傻篮子。

**Reply by frshgrass** [reply]
我觉得做人得自爱 不牵涉什么民族感情 单纯的自尊层面 就得人格独立 （你还是被洗脑了 走向另外的极端 骨子里价值观还是弱肉强食 人能活 动物能活 草也能活 对于各自的生命意义是一样重要的

**Reply by Pacio** [reply]
> 我觉得做人得自爱 不牵涉什么民族感情 单纯的自尊层面 就得人格独立 （你还是被洗脑了 走向另外的极端 骨子里价值观还是弱肉强食 人能活 动物能活 草也能活 对于各自的生命意义是一样重要的

哈哈，笑死~按你的理解和异族结合就是不自爱啊？国内比比皆是的中国女人找老黑，三洞大学陪读（我还收藏了几部小视频你想不想看啊）算了就不发给你了，怕你这种纯粹的中国人受不了

**Comment by TomMao-CN** [comment]
> 我想当新时代的孙中山，大家教教我

鸡蛋从里面打破是生命，从外面打破是食物

**Reply by John-zhan** [reply]
简单类比谬误

**Comment by BurnhamZhang** [comment]
你先混到政治局常委吧，要瓦解也只能从内部瓦解

**Comment by victorchen276** [comment]
为什么要起义？我觉得你们中国人就应该被共匪统治

**Reply by Cegazhos** [reply]
笑嘻了 你是哪个党的幸福子民啊？

**Reply by victorchen276** [reply]
>不像你们中国人都是共匪的子民。哈哈哈哈

**Reply by Cegazhos** [reply]
> > 不像你们中国人都是共匪的子民。哈哈哈哈

你从哪里看到你爹支持D了？

**Comment by ZerodLee** [comment]
要是搞暴力对抗，百分之百会被灭掉，除非你是个难得的奇才，城府极深，信仰及其坚定，通过几十年的努力站到核心高层，然后再卖国求荣，哈哈，你做得到吗？反都不会反的废柴。

**Comment by ghost** [comment]
你个杂种是叫张苏是吧？你等着死无葬身之地吧，已经把你的言论截图发给了有关部门。

**Comment by naweiduxian** [comment]
好家伙Github上还能看到这种言论

**Comment by Ventery** [comment]
这个人说了啥😅 已经看不到了

**Reply by ZVMAGL3** [reply]
他想当孙中山
